AFQ/tasks/data.py

```diff
@@ -219,6 +219,8 @@ def dki_params(brain_mask, gtab, data):
     full path to a nifti file containing
     parameters for the DKI fit
     """
+    if len(dpg.unique_bvals_magnitude(gtab.bvals)) < 3:
+        raise ValueError("Not enough bvals to fit DKI.")
```

Comment:
Also, maybe we can make the error a bit more helpful? Something like: 

```suggestion
        raise ValueError(f"The DKI model requires at least 2 non-zero b-values, but you provided {len(dpg.unique_bvals_magnitude(gtab.bvals))} b-values (including b=0).")
```

Reply:
done!